src/federation-info-plugin/views/SchemaView.jsx

```diff
@@ -12,11 +12,39 @@ const FieldRow = ({ field }) => {
       <td>{input}</td>
       <td>{field.type.toString()}</td>
       <td>{field.ownerServices.join(',')}</td>
-      <td>{field.referencedBy.join(',')}</td>
+      {!noReference && <td>{field.referencedBy.join(',')}</td>}
     </tr>
   )
 }
 
+const FieldsTable = ({ name, fields, noReference }) => {
```

Comment:
can we rename it to `hideReference`, or better work in a positive way and set `showReference`. It's probably more work and is required in more component but working with a `default` value to `false` is easier to mantain